ray-operator/apis/ray/v1/raycronjob_types.go

```diff
@@ -0,0 +1,90 @@
+package v1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type RayJobTemplate struct {
+	// Standard object's metadata of the jobs created from this template.
+	// +optional
+	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
+
+	// Specification of the desired behavior of the job.
+	// +optional
+	Spec RayJobSpec `json:"spec,omitempty" protobuf:"bytes,2,opt,name=spec"`
+}
+
+// CronJobSpec describes how the job execution will look like and when it will actually run.
+type RayCronJobSpec struct {
```

Comment:
Delete `ConcurrencyPolicy` and `Suspend`. It's better to start from `Schedule` and `RayJobTemplate`.